client/reducers/index.js

```diff
@@ -66,6 +66,20 @@ export const getMessagesByStory = (state:Object, storyUid:string) => {
   return R.pickBy(isFromStory, getMessages(state))
 }
 
+export const byDateDescending = R.comparator((d1, d2) => d1.sent > d2.sent)
```

Comment:
Probably don't want to export this (since the name is really generic, but in reality it only works on messages)